littlelines/otwarchive
# Ao3 5884 users

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-5884

## Purpose
Adds restrictions around user management for admins.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended? (If you have access, please copy this into the JIRA ticket for them!)
Sign in as an admin with no roles, and visit the pages referenced in this ticket-- https://otwarchive.atlassian.net/browse/AO3-5884 -- and ensure that admin cannot access the pages they are not supposed to.

Sign in as an admin with the superadmin or policy_and_abuse role, and ensure you can access these pages and the appropriate functions work for this admin.

## References

Are there any other relevant issues / PRs / mailing lists discussions related to this?

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

(If you have an account on JIRA, please include the same name in the "Full name" field on your JIRA profile,
so we can find you and assign you the issues you're working on.)

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

(If you do not fill this in, we will use your GitHub account name and will use "they" to avoid making assumptions about your gender.)
